packages/server/graphql/executeGraphQL.ts

```diff
@@ -48,7 +48,7 @@ const executeGraphQL = async (req: GQLRequest) => {
   } = req
   // never re-use a dataloader since the things it cached may be old
   const dataLoader = getDataLoader(dataLoaderId)
-  dataLoader.share()
+  dataLoader?.share()
```

Comment:
You should fix the type here instead:
https://github.com/ParabolInc/parabol/blob/3e4e15427df9b9b03d3de2a79ab0bd70e4663281/packages/server/graphql/DataLoaderCache.ts#L46-L49